package.json

```diff
@@ -23,7 +23,7 @@
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-react": "^7.12.1",
     "@babel/preset-typescript": "~7.12.7",
-    "@flybywiresim/eslint-config": "~0.1.0",
+    "@flybywiresim/eslint-config": "^0.2.3",
```

Comment:
I just realized we don't have a lock file 🤔

In any case, I would suggest sticking with the `~`-style dependency, then it wil only upgrade to 0.2.4, 0.2.5... but not to 0.3.0. That seems safer.

Reply:
From npm docs:

![image](https://user-images.githubusercontent.com/4503241/141664671-65888633-a9dd-4b55-8d92-99b75b823974.png)

TL;DR: `^` does not allow minor version updates when major version is 0.